src/index.html

```diff
@@ -4,65 +4,375 @@
+            <option value=".bratskaya">14 Days <br> x2</option>
+            <option value=".lesi-ukrainki">28 Days<br> x3</option>
+            <option value=".lesi-ukrainki">56 Days <br> x4</option>
+            <option value=".lesi-ukrainki">112 Days <br> x5</option>
+            <option value=".lesi-ukrainki">224 Days <br> x6</option>
+          </select>
+        </div>
+        <div style="width: 100%; text-align: center;">
+          <!-- <button class="delegate-button">Vote</button> -->
+          <button id="myBtnW" class="delegate-button">Vote</button>
+        </div>
+      </div>
+    </div>
+
+  </div>
+
+  
+
+  </div>
+
+
+
+
+
+
+
+
+ 
+    
+
+    <!-- <section>
         <div class="container">
             <div class="columns is-centered">
                 <div class="column is-narrow">
                     <button class="button is-success is-large">Submit transaction</button>
                 </div>
             </div>
         </div>
-    </section>
+    </section> -->
+
+    <script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js'></script>
     <script type="module">
         import '/src/app.js';
+
+
+        function preLoad() {
+          a1 = new Image; a1.src = 'logos/sik.png';  
+          a2 = new Image; a2.src = 'logos/lucky.png';
+        }
+        function im(image) {
+          document.getElementById(image[0]).src = eval(image + ".src")
+        }
+
+// Get the modal
+var modalN = document.getElementById("myModalN");
+var modalD = document.getElementById("myModalD");
+var modalW = document.getElementById("myModalW");
+
+// Get the button that opens the modal
+var btnN = document.getElementById("myBtnN");
+var btnD = document.getElementById("myBtnD");
+var btnW = document.getElementById("myBtnW");
+
+// Get the <span> element that closes the modal
+var span = document.getElementsByClassName("close")[0];
+// When the user clicks the button, open the modal 
+btnN.onclick = function () {
+  modalN.style.display = "block";
+}
+
+btnD.onclick = function () {
+  modalD.style.display = "block";
+}
+
+btnW.onclick = function () {
+  modalW.style.display = "block";
+}
+
+// When the user clicks on <span> (x), close the modal
+span.onclick = function () {
+  modalN.style.display = "none";
+  modalD.style.display = "none";
+  modalW.style.display = "none";
+}
+
+// span.onclick = function() {
+
+// }
+// When the user clicks anywhere outside of the modal, close it
+window.onclick = function (event) {
+  if (event.target == modalN) {
+    modalN.style.display = "none";
+  } else if (event.target == modalD) {
+    modalD.style.display = "none";
+  } else if (event.target == modalW) {
+    modalW.style.display = "none";
+  }
+}
+
+// window.onclick = function(event) {
+//   if (event.target == modalD) {
+//     modalD.style.display = "none";
+//   }
+// }
+
+
+
+
+
+$('select.select-club-services').each(function () {
+
+  var dropdown = $('<div />').addClass('select-club-services selectDropdown');
```

Comment:
Not a frontend dev, but maybe using `lit.js` (which is already in the project) with custom tags would be better and less error-prone than creating html by hand?
Alternatively, wouldn't it be faster and also easier to test/troubleshoot if you just create all html upfront and only trigger it `enabled` status via CSS and a tiny sparkle of JS?